Mathlib/RingTheory/IntegralClosure/IsIntegral/Basic.lean

```diff
@@ -189,7 +189,7 @@ theorem isIntegral_iff_isIntegral_closure_finite {r : B} :
 @[stacks 09GH]
 theorem fg_adjoin_of_finite {s : Set A} (hfs : s.Finite) (his : ∀ x ∈ s, IsIntegral R x) :
     (Algebra.adjoin R s).toSubmodule.FG :=
-  Set.Finite.induction_on hfs
+  Set.Finite.induction_on _ hfs
```

Comment:
Are we sure we want the `s` argument explicit? It doesn't matter much either way, but if we're changing lots of callers we may as well decide this at the same time.